src/core/openthread.cpp

```diff
@@ -66,9 +66,6 @@ static otDEFINE_ALIGNED_VAR(sInstanceRaw, sizeof(otInstance), uint64_t);
 otInstance *sInstance = NULL;
 #endif
 
-void OT_CDECL operator delete(void *, size_t) throw() { }
-void OT_CDECL operator delete(void *) throw() { }
```

Comment:
The Travis build is passing a bare-metal builds which require definition of delete. This is not a fully representative case. As you see yourself, to build Windows you need to implement those functions. There are other cases, where the implementation of those functions can come from other sources; then we ended up with duplicate definitions.

Since these deletes are only needed for Windows, I propose to move them to windows examples.